src/main/java/Exceptions/ChatAlreadyExistsException.java

```diff
@@ -0,0 +1,20 @@
+package Exceptions;
+
+import java.lang.RuntimeException;
+import org.springframework.http.HttpStatus;
+
+public class ChatAlreadyExistsException extends RuntimeException {
+
+    public static HttpStatus httpStatus = HttpStatus.CONFLICT;
+    public static String message = "Chat Already Exists";
+    public String path;
```

Comment:
These should be final. Other classes should not be able to modify these.

Reply:
Done